implementation/configuration/src/configuration_impl.cpp

```diff
@@ -2286,41 +2286,41 @@ void configuration_impl::load_eventgroup(
                 }
                 its_converter >> its_eventgroup->id_;
             } else if (its_key == "is_multicast") {
-                std::string its_value(j->second.data());
-                if (its_value == "true") {
+                std::string its_value_inner(j->second.data());
+                if (its_value_inner == "true") {
                     its_eventgroup->multicast_address_ = _service->multicast_address_;
                     its_eventgroup->multicast_port_ = _service->multicast_port_;
                 }
             } else if (its_key == "multicast") {
                 try {
-                    std::string its_value = j->second.get_child("address").data();
-                    its_eventgroup->multicast_address_ = its_value;
-                    its_value = j->second.get_child("port").data();
-                    its_converter << its_value;
+                    std::string its_value_inner = j->second.get_child("address").data();
+                    its_eventgroup->multicast_address_ = its_value_inner;
+                    its_value_inner = j->second.get_child("port").data();
+                    its_converter << its_value_inner;
                     its_converter >> its_eventgroup->multicast_port_;
                 } catch (...) {
                     // intentionally left empty
                 }
             } else if (its_key == "threshold") {
                 int its_threshold(0);
-                std::stringstream its_converter;
-                its_converter << std::dec << its_value;
-                its_converter >> std::dec >> its_threshold;
+                std::stringstream its_converter_inner;
+                its_converter_inner << std::dec << its_value;
+                its_converter_inner >> std::dec >> its_threshold;
```

Comment:
we can remove this because it is not needed anymore

Reply:
this is the suggested change:
```suggestion
                its_converter << std::dec << its_value;
                its_converter >> std::dec >> its_threshold;
```